proto/streamlit/proto/Block.proto

```diff
@@ -79,5 +80,12 @@ message Block {
     AvatarType avatar_type = 3;
   }
 
-  // Next ID: 10
+  message Dialog {
+    string title = 1;
+    bool dismissible = 2;
+    optional string width = 3;
```

Comment:
nit: Maybe add a comment here why this needs to be a string since it is a bit different to how width is typed in a few other components.

Reply:
Good point! I will add a comment